command/genesis/genesis.go

```diff
@@ -77,6 +77,15 @@ func setFlags(cmd *cobra.Command) {
 		"the burn contract block and address (format: <block>:<address>[:<burn destination>])",
 	)
 
+	cmd.Flags().StringVar(
+		&params.baseFeeConfig,
+		genesisBaseFeeConfigFlag,
+		command.DefaultGenesisBaseFeeConfig,
+		`initial base fee(in wei) and baseFeeEM (format: <baseFee>[:<baseFeeEM>]). 
```

Comment:
@stana-miric Do you suggest that if a user provides an empty string as value to `genesis-base-fee-config` flag than we should use default values instead of throwing error?

That check in `validateGenesisBaseFeeConfig` is for throwing error when uses gives a empty string as input.

Reply:
```suggestion
		`initial base fee(in wei) and base fee elasticity multiplier (provided in the following format: <baseFee>[:<baseFeeEM>]). 
```